src/client/processor/npc/DueyProcessor.java

```diff
@@ -223,32 +224,97 @@ public static void deleteItem(MapleClient c, int packageId){
         c.announce(MaplePacketCreator.removeItemFromDuey(true, packageId));
     }
     
-     private static void addMesoToDB(int mesos, String sName, int recipientID) {
-        addItemToDB(null, 1, mesos, sName, recipientID);
+    // \/\/\/\/\/\/\/ METHODS FOR API \/\/\/\/\/\/\/\/
+    
+    // Sends a non-equipment item to an account
+    public static void sendItem(int itemId, int quantity, int mesos, String sender, int recvAcctId){
+        sendItem(itemId, quantity, mesos, -1, sender, 0, recvAcctId);
```

Comment:
what happens when you try to send both item and mesos at the same time?

Reply:
Two packages will be sent, one with the item and one with only mesos.

Reply:
looks good. I will likely be using this method for server's api call.